server/src/main/scala/de/frosner/broccoli/services/InstanceService.scala

```diff
@@ -236,99 +249,115 @@ class InstanceService @Inject()(nomadClient: NomadClient,
 
   def updateInstance(id: String,
                      statusUpdater: Option[JobStatus],
-                     parameterValuesUpdater: Option[Map[String, String]],
+                     parameterValuesUpdater: Option[Map[String, JsValue]],
```

Comment:
We use `Map[String, JsValue]` in some places and `Map[String, ParameterValue]` in others. Why is that?

Reply:
We use JsValue until the point where we can actually decode it. (see `InstanceService.updateInstance`). Once we have the information (from the template.parameterinfo) we can parse it accordingly.